### PR TITLE
Added cacert file and ssl verification to http client.

### PR DESCRIPTION
**Pull request by DanayaMel**
This PR adds a cacert.pem file needed for Guzzle to make successful ssl curl calls. The file was downloaded from here: https://curl.haxx.se/docs/caextract.html. It also adds a `validationUrl` getter to the ApplePayAuthorizeRequest class.

### CODE REVIEW

**Comment by JCManzo** [comment]
> It also adds a `validationUrl` getter to the ApplePayAuthorizeRequest class.

I'm not seeing this as a change for this PR

File: src/Message/ApplePayAuthorizeRequest.php [hunk 1]
```diff
@@ -358,6 +360,7 @@ protected function createClientRequest($data, array $headers = null)
 
         $config->set('curl.options', $curlOptions);
         $this->httpClient->setConfig($config);
+        $this->httpClient->setDefaultOption('verify', __DIR__ . '/../../certs/cacert.pem');
```
Comment (JCManzo) [inline src/Message/ApplePayAuthorizeRequest.php]: Rather than hard-coding this cert dependency and comiting the cert file manually what do you think about managing this via composer. For example with [composer/ca-bundle](https://github.com/composer/ca-bundle) and then you can use it with Guzzle like so https://github.com/composer/ca-bundle#to-use-with-guzzle

**Comment by DanayaMel** [comment]
> > It also adds a `validationUrl` getter to the ApplePayAuthorizeRequest class.
> 
> I'm not seeing this as a change for this PR. I'm reading this as you added a `validationUrl()` method.

Sorry, I worded that poorly. I meant a validationUrl getter. So the function is `getValidationUrl()` –– found [here](https://github.com/vimeo/omnipay-vindicia/pull/50/files#diff-ba17822789aa8e485b85ea6429963389R327).

File: src/Message/ApplePayAuthorizeRequest.php [hunk 2]
```diff
@@ -324,8 +324,9 @@ public function getData()
         $data = array();
 
         // Required request parameters for Apple Session object.
-        $data['merchantIdentifier'] = $this->getMerchantIdentifier();
+        $data['validationUrl'] = $this->getValidationUrl();
```
Comment (nickyr) [inline src/Message/ApplePayAuthorizeRequest.php]: is this related to the cert at all? if not, could you make this change a separate PR? not clear what this is for

File: src/Message/ApplePayAuthorizeRequest.php [hunk 3]
```diff
@@ -340,6 +341,7 @@ public function getData()
      * @param array $headers
      *
      * @return static
+     * @psalm-suppress UndefinedMethod
```
Comment (nickyr) [inline src/Message/ApplePayAuthorizeRequest.php]: could you add a comment saying why?

File: src/Message/ApplePayAuthorizeRequest.php (hunk 1 above)
Comment (nickyr) [reply src/Message/ApplePayAuthorizeRequest.php]: oh that's awesome, good idea @JCManzo!

Comment (nickyr) [reply src/Message/ApplePayAuthorizeRequest.php]: also yay @DanayaMel for figuring out how to do this with guzzle functions!

**Comment by DanayaMel** [comment]
@nickyr @JCManzo changes made

File: src/Message/ApplePayAuthorizeRequest.php [hunk 4]
```diff
@@ -340,6 +340,7 @@ public function getData()
      * @param array $headers
      *
      * @return static
+     * @psalm-suppress UndefinedMethod because Guzzle\Client::setDefaultOption() is actually defined.
```
Comment (nickyr) [inline src/Message/ApplePayAuthorizeRequest.php]: lol do you know why it thinks it's not?
